### PR TITLE
Levinson update

### PR DESCRIPTION
**Pull request by nooges**
## Description

Move away from using common rules.mk file to revision specific ones. Also update RGB wiring method to use `RGBLED_SPLIT` so that both halves are wired the same way.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by zvecr** [comment]
Without a top level rules.mk having a DEFAULT_FOLDER and code files at the top level, it makes CI really difficult. For instance, if `levinson/config.h` changes, we have no way to tie together what should be built. Other boards follow this pattern, and you can see within #7368 how I've had to temporarily work round the issue for the iris.

TLDR is that i dont think we should ditch the top level rules.mk if there are still code files there.

**Comment by nooges** [comment]
So `planck` doesn’t have a top level rules.mk and does have code, which is what I’ve modeled the changes to Levinson after (and iris/Nyquist previously).

**Comment by zvecr** [comment]
Yeah doesnt mean that one is "correct" either....Ive flagged it as an issue where ive noticed the changes. I get that it stops users from flashing the wrong firmware but we are really just trading one issue for another.

For lack of a better term, if this is now a common standard, then i think its time to accept a CI dependent bodge.

File: keyboards/keebio/levinson/rev2/config.h [hunk 1]
```diff
@@ -51,8 +51,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 /* ws2812 RGB LED */
 #define RGB_DI_PIN D3
-
 #define RGBLED_NUM 12    // Number of LEDs
+#define RGBLED_SPLIT { 6, 6 }
```
Comment (zvecr) [inline keyboards/keebio/levinson/rev2/config.h]: From <https://docs.keeb.io/adding-rgb-underglow/> where the ws2812 signal is passed over trrs, won't this break existing users?

File: keyboards/keebio/levinson/readme.md [hunk 2]
```diff
@@ -7,21 +7,25 @@ Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)
...
+
 Make example for this keyboard (after setting up your build environment):
 
-    make levinson/rev2:default
+    make keebio/levinson/rev2:default
 
 Example of flashing this keyboard with Pro micro:
 
-    make levinson/rev2:default:avrdude
+    make keebio/levinson/rev2:default:avrdude
```
Comment (drashna) [inline keyboards/keebio/levinson/readme.md]:
```suggestion
    make keebio/levinson/rev2:default:flash
```

File: keyboards/keebio/levinson/readme.md [hunk 3]
```diff
@@ -7,21 +7,25 @@ Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)
...
 
 Example of flashing this keyboard with Elite-C:
 
 You'll need to add `BOOTLOADER = atmel-dfu` to your `rules.mk` to ensure that resetting the board works properly.
 
 Hold the reset button on the Elite-C board, or the reset button on the keyboard if already assembled, then release the button and run:
 
-    make levinson/rev2:default:dfu
+    make keebio/levinson/rev2:default:dfu
```
Comment (drashna) [inline keyboards/keebio/levinson/readme.md]:
```suggestion
    make keebio/levinson/rev2:default:flash
```

File: keyboards/keebio/levinson/rev1/rules.mk [hunk 4]
```diff
@@ -1 +1,36 @@
...
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight.
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
+
+SPLIT_KEYBOARD = yes
+
+LAYOUTS = ortho_4x12
```
Comment (drashna) [inline keyboards/keebio/levinson/rev1/rules.mk]:
```suggestion
LAYOUTS = ortho_4x12
LAYOUT_HAS_RGB = yes
```

File: keyboards/keebio/levinson/rev2/config.h (hunk 1 above)
Comment (drashna) [reply keyboards/keebio/levinson/rev2/config.h]: Yeah, it may

File: keyboards/keebio/levinson/rev3/rules.mk [hunk 5]
```diff
@@ -1 +1,36 @@
...
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight.
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
+
+SPLIT_KEYBOARD = yes
+
+LAYOUTS = ortho_4x12
```
Comment (drashna) [inline keyboards/keebio/levinson/rev3/rules.mk]:
```suggestion
LAYOUTS = ortho_4x12
LAYOUT_HAS_RGB = yes
```
